packages/darklang/languageTools/parser.dark

```diff
@@ -30,6 +30,12 @@ module Darklang =
       let parseToSimplifiedTree (text: String) : ParsedNode =
         Builtin.Parser.parseToSimplifiedTree text
 
+      let parseName (fnName: String) : Darklang.Stdlib.Result.Result<String> =
+        if (Darklang.Stdlib.String.contains fnName "_v") then
+          // Todo : check if the name is valid
+          Darklang.Stdlib.Result.Result.Ok fnName
+        else
+          Darklang.Stdlib.Result.Result.Ok(fnName ++ "_v0")
```

Comment:
I'm curious why we need this. Is there something that expects the _v0 suffix to be present?

Reply:
Yes, I believe this doesn't work without the _v0
`http://dark-packages.dlio.localhost:11003/function/by-name/Darklang.Stdlib.String.toBytes_v0`

It fails with 
`PACKAGE.Darklang.Stdlib.Int64.ParseError.BadFormat`